source/clear-linux/guides/maintenance/swupd-mixer-integration.rst

```diff
@@ -0,0 +1,155 @@
+.. _swupd-mixer-integration:
+
+Mixer tool enhancement
+######################
+
+Mixing means composing an operating system for a specific use case. The
+default build of |CLOSIA| provides options to install bundles for various
+server capabilities. If you want to supplement the default OS with
+functionality from other OS distributions, you can manually compose an OS
+using :ref:`the mixer tool <mixer>`.
+
+Software updater release 19900 integrates the mixer tool with the
+updater (:command:`swupd`) and provides a simpler way to mix your OS.
+
+Process overview
+****************
+
+To add a package that does not exist in the current |CL| bundle, perform the following:
+
+#. Create custom packages and bundles using the :file:`swupd-add-pkg` script.
```

Comment:
This is wrong, swupd-add-pkg "adds" a package it does not create it. Building packages is an entirely different (and complex enough) topic on its own, we can link to it : https://github.com/clearlinux/common/blob/master/README.md